elasticdl/python/common/args.py

```diff
@@ -261,6 +261,15 @@ def add_common_params(parser):
         "the job automatically, and users need to launch the job through "
         "command `kubectl create -f path_to_yaml_file`.",
     )
+    add_bool_param(
+        parser=parser,
+        name="--force_use_kube_config_file",
```

Comment:
`use_kube_config_file` should be enough.

Reply:
if use_kube_config_file == False，we will also load the kube config file if the pod is not running in a K8S environment.
IMO, add `force` may be clear